tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3753 @@
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '404':
+          description: "Unsuccessful operation: indicates that the requested data was not found."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFoundResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+  
+  '/api/core/v3/commitments/by-index/{index}':
+    get:
+      tags:
+        - commitments
+      summary: Look up a commitment by a given commitment index.
+      description: Look up a commitment by a given commitment index.
+      parameters:
+        - in: path
+          name: index
+          schema:
+            type: integer
+          example: 154862
+          required: true
+          description: Index of the commitment to look up.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Commitment'
+              examples:
+                default:
+                  $ref: >-
+                    #/components/examples/get-commitment-response-example
+            application/vnd.iota.serializer-v2:
+              schema:
+                type: string
+                format: binary
+                description: Commitment in raw binary format
+              examples:
+                default:
+                  $ref: >-
+                    #/components/examples/get-commitment-response-binary-example
+        '400':
+          description: "Unsuccessful operation: indicates that the provided data is invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '404':
+          description: "Unsuccessful operation: indicates that the requested data was not found."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFoundResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+  '/api/core/v3/commitments/by-index/{index}/utxo-changes':
```

Comment:
Missing the full route for this: `'/api/core/v3/commitments/by-index/{index}/utxo-changes/full'`

Reply:
Yes, let's add it.

Reply:
Should be `by-slot` and not `by-index`

Reply:
Oh!!  Yeah, by-slot is more precise.